MultipleTopLevelWebVIs/Readme.md

```diff
@@ -42,9 +42,12 @@ Copy and paste the entire `MultipleTopLevelVIs` directory built under `<Project
 # Details
 
 ## File Name and Relative URL
-Each of the top-level WebVIs generates a corresponding HTML file. The organization of the generated HTML files is determined by the order of the top-level WebVIs in the `WebApp.gcomp` file.
+Each of the top-level WebVIs generates a corresponding HTML file. The organization of the generated HTML files is determined by the hierarchy of the top-level WebVIs in the `WebApp.gcomp` file.
```

Comment:
Gah sorry, this still feels like it could be confusing to a user, hmm...

Maybe it is clearer as `the hierarchy of WebVIs marked as top-level in the WebApp.gcomp`?

I feel like a user may confuse "top-level" the property that can be ticked on any arbitrary WebVI with top-level as in a WebVI in the root of the namespace which is not what's trying to be conveyed.